### ISSUE TITLE
Test Case Failed

### ISSUE THREAD

**Issue by MuhammetDilmac** [EvanHahn/crystal-helmet#1]
Hello,
When i run crystal spec i get an error.
Error;
```
Error in line 1: while requiring "./spec/nosniffhandler_spec.cr"

in ./spec/nosniffhandler_spec.cr:9: no overload matches 'Helmet::NoSniffHandler#next=' with type TestNext
Overloads are:
 - HTTP::Handler#next=(next : Handler | Proc | Nil)

    handler.next = next_handler
            ^~~~
```

**Comment by EvanHahn** [comment]
Hmm, perhaps Crystal has changed.

Any chance you'd be able to take a stab at a pull request?

**Comment by MuhammetDilmac** [comment]
Hi @EvanHahn,
I don't understand to why you use TestNext class?

**Comment by EvanHahn** [comment]
I probably missed it! I can take a look at this when I get to it. If you would like to make a pull request, feel free to!

**Comment by bthachdev** [comment]
I made a pull request to fix this problem.
https://github.com/EvanHahn/crystal-helmet/pull/3

**Comment by EvanHahn** [comment]
Thanks to #3, this has been fixed in version 0.2.1.
